.github/workflows/publish.yml

```diff
@@ -18,10 +18,17 @@ jobs:
         with:
           arguments: modrinth
           gradle-home-cache-cleanup: true
-      - name: Archive Artifacts
+      - name: Archive Artifacts (fabric)
```

Comment:
can we further edit this workflow so that PRs are built and so that no publishing is performed for non-GeyserMC/non-master braches (our forks are failing)

maybe rename it if those changes are applied